I'm sorry, but an uncaught exception occurred.

While loading <renpy.display.im.Image object ('images/bg_puzzle_rotate.png') at 0x0000000005515e50>:
  File "game/script.rpy", line 831, in script
    call screen rotate_batik
ScriptError: could not find label 'story_2'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "D:\RenPy\renpy-8.3.0-sdk\renpy\bootstrap.py", line 359, in bootstrap
    renpy.main.main()
  File "D:\RenPy\renpy-8.3.0-sdk\renpy\main.py", line 652, in main
    run(restart)
  File "D:\RenPy\renpy-8.3.0-sdk\renpy\main.py", line 148, in run
    renpy.execution.run_context(True)
  File "D:\RenPy\renpy-8.3.0-sdk\renpy\execution.py", line 958, in run_context
    context.run()
  File "game/script.rpy", line 831, in script
    call screen rotate_batik
  File "D:\RenPy\renpy-8.3.0-sdk\renpy\script.py", line 1103, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'story_2'.

Windows-10-10.0.19045 AMD64
Ren'Py 8.3.0.24082114
 
Wed Dec  4 02:22:14 2024
